axiom/scripts/axiomatic.py

```diff
@@ -121,11 +121,15 @@ def parseOptions(self, args):
             # If a reactor is being selected, it must be done before the store
             # is opened, since that may execute arbitrary application code
             # which may in turn install the default reactor.
-            if "--reactor" in args:
-                reactorIndex = args.index("--reactor")
-                shortName = args[reactorIndex + 1]
-                del args[reactorIndex:reactorIndex + 2]
-                self.opt_reactor(shortName)
+            for index, arg in enumerate(args):
+                if arg.startswith("--reactor"):
```

Comment:
I'd prefer fixing the bug just because it's so subtle (probably more subtle than the issue we're fixing); even if it's not that likely to come up, if it ever does come up, it's probably going to cause somebody a lot of head scratching.